docs/workshop/onboarding.md

```diff
@@ -6,7 +6,9 @@ In this workshop, you will learn how to collect data from an open CI/CD data sou
 
 ### Access
 
-To get access to the services please add your github username [here](https://docs.google.com/spreadsheets/d/1j-i0O-sbk6WWZpWsMl9uSpUjaeQ5LXEY44iNp-vk2eA/edit?usp=sharing), and you will be provided access.
+In order to get started with the workshop, it's vital to have a github account, so to create your github account please follow along this [documentation](git_setup.md).
+
+To get access to the services please add your github username [here](https://etherpad.opendev.org/p/workshop_scale_2022), and you will be provided access.
 
 
 To get started with this tour, let’s first introduce you to the various tools that are available on Operate First environment and will help us in building the AIOps application using Operate First.
```

Comment:
Can we make this flow easier to follow? And @aakankshaduggal instead of sending people to the git setup document can we copy the first step with the image here and remove it from the git setup doc ? 

I remember @quaid had pointed out that the creating git account at that point of the workshop didnt make sense

<img width="968" alt="Screen Shot 2022-07-25 at 2 39 42 PM" src="https://user-images.githubusercontent.com/32435206/180851313-30e2c46e-3879-47aa-b811-d6162bab8d01.png">

```suggestion
### Step 1: Create Github Account
In order to get started with the workshop, it's vital to have a github account, so please create a github account.

### Step 2: Get added to the cluster

To get access to the services please add your github username [here](https://etherpad.opendev.org/p/workshop_scale_2022), and you will be provided access.

### Step 3: Try out the Services

To get started with this tour, let’s first introduce you to the various tools that are available on Operate First environment and will help us in building the AIOps application using Operate First.
```